### ISSUE TITLE
Crash in md_array_str_eq()

### ISSUE THREAD

**Issue by mkauf** [icing/mod_md#224]
I got a crash in md_array_str_eq():
```
#0  0x00007f0ed9f99bd5 in md_array_str_eq (a1=0x556fc1771ec0, a2=0x0, case_sensitive=case_sensitive@entry=0) at md_util.c:226
#1  0x00007f0ed9f943ea in md_reg_sync_finish (reg=0x556fc1771a30, md=md@entry=0x556fc1719ae0, p=p@entry=0x556fc167f138, ptemp=ptemp@entry=0x556fc16aa3c8) at md_reg.c:877
#2  0x00007f0ed9f9e00e in md_post_config_after_ssl (p=0x556fc167f138, plog=<optimized out>, ptemp=0x556fc16aa3c8, s=<optimized out>) at mod_md.c:949
#3  0x0000556fc1377ff3 in ap_run_post_config (pconf=pconf@entry=0x556fc167f138, plog=0x556fc16ae488, ptemp=0x556fc16aa3c8, s=0x556fc16a9858) at config.c:103
#4  0x0000556fc1352f0e in main (argc=<optimized out>, argv=<optimized out>) at main.c:807
```

Probably you have already fixed it in this commit: https://github.com/icing/mod_md/commit/ebe24fc21d655c92cb97f4e73e9982818cb10f44

It's this change in src/md_util.c:
```
-    if (!a1) return 0;
+    if (!a1 || !a2) return 0;
```

Any chance to backport this bugfix to the branch "maintenance-2.2.x" ? I hope it will be included in a future version of httpd 2.4.x.

**Comment by icing** [comment]
Ah, yes. Should do that. Some people use the module from here, also.

**Comment by icing** [comment]
I will release master (containing the fix) as 2.4.0 soon and bring this over to Apache repository.
